src/linearmixedmodel.jl

```diff
@@ -285,15 +285,28 @@ diagonal blocks from the conditional variance-covariance matrix,
     s² Λ(Λ'Z'ZΛ + I)⁻¹Λ'
 """
 function condVar(m::LinearMixedModel{T}) where {T}
-    retrms = m.reterms
-    t1 = first(retrms)
-    L11 = first(m.L)
-    if !isone(length(retrms)) || !isa(L11, Diagonal{T,Vector{T}})
-        throw(ArgumentError("code for multiple or vector-valued r.e. not yet written"))
+    L = m.L
+    s = sdest(m)
+    spL = LowerTriangular(sparseL(m))
```

Comment:
```suggestion
    # when https://github.com/JuliaLang/julia/pull/40174 lands,
    # we can update the version bound
    @static if VERSION < v"2"
        spL = LowerTriangular(SparseMatrixCSC{T, Int}(sparseL(m)))
    else 
        spL = LowerTriangular(sparseL(m))
    end
```

It adds an extra step, but at a point where it should be relatively cheap, and the type change across versions isn't exposed to the user / doesn't cause type instability in the final result.

Reply:
Do we have a version number to fill in here?  Yesterday it was `v"1.7.0-DEV.761"` which is only three commits after your modification.

Reply:
I think we can use that number. The change is also marked to be backported to 1.6.1 when it comes out.